### PR TITLE
Clean up messages and strings. 

### PR DESCRIPTION
**Pull request by awholegunch**
Removes a confusing example, makes case consistent, and other cleanup.

### CODE REVIEW

**Comment by ajainarayanan** [comment]
:+1: LGTM on UI changes.

File: cdap-ui/templates/cdap-etl-realtime/Stream.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
         },
         "name": {
           "widget": "stream-selector",
-          "label" : "Name",
-          "description" : "Name of the stream to which we write to. If the stream is present, then data will be written to it. If it's not present then one would be created.",
+          "label" : "Stream Name",
```
Comment (bdmogal) [inline cdap-ui/templates/cdap-etl-realtime/Stream.json]: When users see this, they're already configuring a Stream source. Do we need to mention Stream again?

**Comment by bdmogal** [comment]
Quick Question. LGTM otherwise.

File: cdap-ui/templates/cdap-etl-realtime/Stream.json (hunk 1 above)
Comment (awholegunch) [inline cdap-ui/templates/cdap-etl-realtime/Stream.json]: I think so. There are three fields there, all ending in "Name". For some reason, the "Stream Name" is last. It is also consistent with others.

**Pull request merged**